blazor/stock-chart/panning.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Panning in Blazor Stock Chart Component | Syncfusion
+description: Checkout and learn here all about panning functionality in Syncfusion Blazor Stock Chart component and more.
+platform: Blazor
+control: Stock Chart
+documentation: ug
+---
+
+# Panning in Blazor Stock Chart Component 
+
+To ensure that users can start interacting with the chart as soon as it renders, panning is enabled by default in the stock chart. The [EnablePan](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.StockChartZoomSettings.html#Syncfusion_Blazor_Charts_StockChartZoomSettings_EnablePan) property in the [StockChartZoomSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.StockChartZoomSettings.html), which by default has the value **true**, defines this default behavior.
+
```

Comment:
By default, panning is enabled in the Stock Chart, ensuring that users can immediately start interacting with the chart upon rendering. This default behavior is controlled by the [EnablePan](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.StockChartZoomSettings.html#Syncfusion_Blazor_Charts_StockChartZoomSettings_EnablePan) property, which is set to true by default in the [StockChartZoomSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.StockChartZoomSettings.html).


This is itself fine.

Reply:
Changed mam.